src/server/lsp/index.ts

```diff
@@ -33,12 +34,19 @@ export class LspServer extends Dispose {
     return this._client;
   }
 
-  async init() {
+  private lastId = 0;
+  private outchannel?: OutputChannel;
+
+  async init(): Promise<void> {
+    this.outchannel = workspace.createOutputChannel('flutter-lsp');
```

Comment:
Should not recreate `workspace.createOutputChannel('flutter-lsp');` when restart, it will be better to separate the init and start lsp behavior